### DISCUSSION TITLE
turbo mode without assigning keyboard or mouse buttons

### DISCUSSION THREAD

**Discussion by YogSottot** [AntiMicroX/antimicrox#844] [Q&A]
Is there any way to enable turbo mode on controller button, not keyboard/mouse button?

I have a controller enabled game but would like turbo mode or hold for turbo to be assigned to the controller button without a button reassignment.
I have played around with antimicrox for a while and have found no way to ability to insert a controller action (not a keyboard or mouse action).

I would think that checking the turbo option and not assigning the KB/M button would work but it does not.

**Comment by clockwinder** [comment]
@YogSottot I have this exact same question. Did you find a solution?

**Reply by YogSottot** [reply]
@clockwinder 

There is no complete solution.

I only needed this feature to complete the mini-game, not for everyday use.
So I simply assigned the keyboard keys to the gamepad buttons and enabled turbo and toggle mode for them. Then I created an antimicro profile with these settings and completed the mini-game.

**Reply by Jopp-gh** [reply]
The solution is to add a key unused by your controller / kb and to switch to another group, not profile. (Groups are at the bottom of AntiMicroX's interface)

You've to enable one of your gamepad buttons to switch to another group back and forth.
